app/src/main/java/org/fossasia/pslab/communication/sensors/BH1750.java

```diff
@@ -0,0 +1,58 @@
+package org.fossasia.pslab.communication.sensors;
+
+import org.fossasia.pslab.communication.peripherals.I2C;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+
+public class BH1750{
+    private String TAG = "BH1750";
+    private int POWER_ON = 0x01;
+    private int RESET = 0x07;
+    private int RES_1000mLx = 0x10;
+    private int RES_500mLx = 0x11;
+    private int RES_4000mLx = 0x13;
+    private I2C i2c;
+
+    public int[] gain_choices = {RES_500mLx,RES_1000mLx,RES_4000mLx};
+    public String[] gain_literal_choices = {"500mLx", "1000mLx", "4000mLx"};
+    public int gain = 0;
+    public double[] scaling = {2,1,0.25};
+
+    private HashMap<String, ArrayList> params = new HashMap<>();
+
```

Comment:
params is unused here.
params in python communication library is used to generate GUI efficiently. We have yet to discuss implementation details about how to use this params data to generate GUI in Android dynamically instead of Hard-coding all the sensors UI.
If you have a possible implementation. Feel free to discuss it.